src/content/content.service.ts

```
@@ -288,7 +288,7 @@ export class ContentService {
 			if (response) {
 				// Log number of BPP responses received
 				const bppCount = response.responses?.length || 0;
-				//const bppIds = response.responses?.map(r => r.context?.bpp_id).filter(Boolean) || [];
+				const bppIds = response.responses?.map(r => r.context?.bpp_id)|| [];
```

ESLint

[error] Replace `·response.responses?.map(r·=>·r.context?.bpp_id)` with `⏎↹↹↹↹↹response.responses?.map((r)·=>·r.context?.bpp_id)·`

(prettier/prettier)